doc/cla/corporate/syncordia.md

```diff
@@ -0,0 +1,16 @@
+Ireland, 2015-06-26
+
+Syncordia Technologies and Healthcare Solutions Ireland Limited agrees to the terms of the 
+Odoo Corporate Contributor License Agreement v1.0.
+
+I declare that I am authorized and able to make this agreement and sign this
+declaration.
+
+Signed,
+
+Vadim Zenin vadim.zenin@syncordiahealth.ie https://github.com/Vadim-Zenin
+
+List of contributors:
+
+Vadim Zenin vadim.zenin@syncordiahealth.ie https://github.com/Vadim-Zenin
+Vadims Briksins vadim.bryksin@syncordiahealth.ie https://github.com/Briksins
```

Comment:
@Briksins The thing is, it's not a fix, it's an improvement/feature. Hence the target for Odoo 9, as a general policy. If we start accepting everybody's wishlist in stable series we drastically increase the risk of regressions and side-effects, in a series that is in production in thousands of companies. Many companies periodically update to the latest stable revision without specifically re-testing everything (that's what *stable* is about), so the cost of any side-effect is very high. The "improvements" are simply not worth the risk in general, so our [policy](https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean) is quite clear about that. When included in the next major version, these improvements will have a lot more battle-testing and will only be put in production after a proper upgrade/staging process. Thanks again for your contributions and your quick answers.